src/Xamarin.Android.Cecil/Xamarin.Android.Cecil.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <CecilDirectory>$(MSBuildThisFileDirectory)\..\..\external\cecil</CecilDirectory>
+    <CecilPreparedFlag>prepared.flag</CecilPreparedFlag>
+    <OutputPath Condition="'$(OutputPath)' == ''">bin\$(Configuration)</OutputPath>
+    <CecilOutputPath>$([System.IO.Path]::GetFullPath ('$(OutputPath)'))</CecilOutputPath>
+  </PropertyGroup>
+  <Target Name="PrepareCecil"
+      Inputs="$(CecilDirectory)\Mono.Cecil.sln;AssemblyInfo.cs;Mono.Cecil.overrides;Xamarin.Android.Cecil.targets"
+      Outputs="$(CecilPreparedFlag)">
+    <Copy SourceFiles="AssemblyInfo.cs" DestinationFiles="$(CecilDirectory)\Mono.Cecil.AssemblyInfo.cs" />
+    <Copy SourceFiles="Mono.Cecil.overrides" DestinationFolder="$(CecilDirectory)" />
+    <Exec Command="cd $(CecilDirectory); git reset --hard" />
```

Comment:
This shouldn't be necessary.

Instead, we should add a top-level `make prepare` target, a'la [`xamarin-android`](https://github.com/xamarin/xamarin-android/blob/a9bd478/Makefile#L30-L40), and should also contain/handle the [`nuget restore`](https://github.com/xamarin/java.interop/blob/f915754/Makefile#L63) logic.